src/main/kotlin/solve/importer/view/AlertDialog.kt

```diff
@@ -0,0 +1,54 @@
+package solve.importer.view
+
+import javafx.geometry.Insets
+import javafx.geometry.Pos
+import javafx.scene.image.ImageView
+import javafx.scene.layout.BorderPane
+import javafx.scene.text.Font
+import kotlinx.coroutines.cancel
+import solve.constants.IconsAlertError
+import solve.utils.loadResourcesImage
+import solve.utils.mfxButton
+import tornadofx.*
+
+class AlertDialog(contentText: String) : View() {
```

Comment:
Uncatched exception when invalid directory is selected
![IzkhfnssF0](https://user-images.githubusercontent.com/55746901/232251905-65ee27c0-db71-4fbe-97ea-abf2da6e1e6e.gif)